protocol/communities/manager.go

```diff
@@ -1200,7 +1200,7 @@ func (m *Manager) InviteUsersToCommunity(communityID types.HexBytes, pks []*ecds
 	return m.inviteUsersToCommunity(community, pks)
 }
 
-func (m *Manager) AddMemberToCommunity(communityID types.HexBytes, pk *ecdsa.PublicKey) (*Community, error) {
+func (m *Manager) AddMemberAdminToCommunity(communityID types.HexBytes, pk *ecdsa.PublicKey) (*Community, error) {
```

Comment:
Don't we still want to keep the `AddMemberToCommunity()` API around?

Reply:
It is used only in ImportCommunity(). If it was used in a couple of places also for adding regular members I would leave it. Now, I think that maybe Owner (instead of Admin) will be more correct word, so probably I will rename it.